requirements/base.txt

```diff
@@ -122,8 +122,6 @@ geographiclib==1.52
     # via geopy
 geopy==2.2.0
     # via apache-superset
-greenlet==2.0.2
-    # via sqlalchemy
```

Comment:
We'd really need to add linting to these requirements to make sure they're up to date. The python lint workflow would be perfect for this.

Reply:
+1